src/org/labkey/test/util/TestDataGenerator.java

```diff
@@ -582,7 +582,8 @@ public static String randomFieldName(String part, int numStartChars, int numEndC
     public static String randomFieldName(@NotNull String part, int numStartChars, int numEndChars, @Nullable String exclusion)
     {
         // use the characters that we know are encoded in fieldKeys plus characters that we know clients are using
-        String chars = ALL_ILLEGAL_QUERY_KEY_CHARACTERS + " %()=+-[]_|*`'\":;<>?!@#^" + NON_LATIN_STRING + WIDE_PLACEHOLDER ;
+        // Issue 53197: Field name with double byte character can cause client side exception in Firefox when trying to customize grid view.
+        String chars = ALL_ILLEGAL_QUERY_KEY_CHARACTERS + " %()=+-[]_|*`'\":;<>?!@#^" + NON_LATIN_STRING;// + WIDE_PLACEHOLDER ;
```

Comment:
This is a bit of a shame, because it is finding issues. Isn't it worth the noise?
If you are sure this is the best option you should add a comment to issue 53197 to add back the `WIDE_PLACHOLDER` once the bug is fixed.